pythonforandroid/graph.py

```diff
@@ -51,14 +52,15 @@ def get_dependency_tuple_list_for_recipe(recipe, blacklist=[]):
 
         # Filter out blacklisted items and turn lowercase:
         dependencies = [
-            deptuple for deptuple in dependencies
-            if not set(deptuple).intersection(set(blacklist))
+            tuple(set(deptuple) - blacklist)
+            for deptuple in dependencies
+            if tuple(set(deptuple) - blacklist)
```

Comment:
Minor/troll: so you prefer to use `intersect()` over `&` but still want to use `-` over `difference()` :thinking: 
Honestly I prefer to use the operator for all operations